src/core/BufferGeometry.js

```diff
@@ -636,28 +636,26 @@ BufferGeometry.prototype = Object.assign( Object.create( EventDispatcher.prototy
 				box.setFromBufferAttribute( position );
 				box.getCenter( center );
 
-				// hoping to find a boundingSphere with a radius smaller than the
-				// boundingSphere of the boundingBox: sqrt(3) smaller in the best case
-
 				var maxRadiusSq = 0;
 
-				for ( var i = 0, il = position.count; i < il; i ++ ) {
+				var array = position.array;
+				var count = array.length;
 
-					vector.x = position.getX( i );
-					vector.y = position.getY( i );
-					vector.z = position.getZ( i );
-					maxRadiusSq = Math.max( maxRadiusSq, center.distanceToSquared( vector ) );
+				for ( var i = 0; i < count; i += 3 ) {
 
-				}
+					vector.set( array[ i ], array[ i + 1 ], array[ i + 2 ] );
```

Comment:
We use methods like `.getX()`  so stuff like `InterleavedBufferAttribute` also works. Direct array access breaks this support.